tests/run.sh

```diff
@@ -27,3 +28,21 @@ wait_for_ingress_ready 'knative-ingressgateway' 'istio-system'
 kubectl create namespace $NAMESPACE
 fats_create_push_credentials $NAMESPACE
 riff namespace init $NAMESPACE $NAMESPACE_INIT_FLAGS
+
+# run test functions
+echo "Run functions"
+source `dirname "${BASH_SOURCE[0]}"`/../functions/helpers.sh
+
+# uppercase
+for test in java java-boot java-local node npm command; do
+  path=`dirname "${BASH_SOURCE[0]}"`/../functions/uppercase/${test}
```

Comment:
README says files for functions can be "anywhere". Maybe it's ok to force location and auto-discover them?

Reply:
we could do that, but it would seem to make functions like `run_function` with currently accept a path less flexible. Consumers of fats would need to move their function into the proper directory and then. Clusters and registries have a more complex lifecycle that needs to be sourced in a predictable way. I expect functions to be a more common extension point than clusters or registries.

I'm open to making the location more opinionated if you think the tradeoff is worthwhile.